xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrUserFacet.xml

```diff
@@ -42,7 +42,7 @@
   #set ($escapedUserName = $escapetool.xml($xwiki.getPlainUserName($userReference)))
   #getUserAvatarURL($userReference $avatarURL 30)
   &lt;span class="user"&gt;##
-    &lt;img class="user-avatar" src="$escapetool.xml($avatarURL.url)" alt="" /&gt;##
+    &lt;img class='user-avatar' src="$escapetool.xml($avatarURL.url)" alt='' /&gt;##
```

Comment:
This change looks unrelated.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2187/commits/347fb7f510c1139738e466662fa735b9d5db43f0